data/json/items/tool/musical_instruments.json

```
@@ -308,7 +311,7 @@
     "category": "tools",
     "weight": "3750 g",
     "color": "red",
-    "to_hit": 2,
+    "to_hit": { "grip": "none", "length": "short", "surface": "any", "balance": "clumsy" },
```

I think you committed the wrong one.
```suggestion
    "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "clumsy" },
```